extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.tracking.js

```diff
@@ -121,6 +121,7 @@ define('wikia.articleVideo.featuredVideo.tracking', ['wikia.tracker'], function
 						{ label: 'user-start' };
 				}
 
+				playerInstance.trigger('firstVideoPlay');
```

Comment:
Event name suggests it's fired only for the first video. I believe this event is fired on video start. Can we rename this event?